lighty-modules/lighty-gnmi/lighty-gnmi-test/src/test/java/io/lighty/modules/gnmi/test/gnmi/GnmiWithoutRestconfTest.java

```diff
@@ -287,6 +287,13 @@ public void testCrudOperation() throws ExecutionException, InterruptedException,
         } catch (ExecutionException | InterruptedException e) {
             Assertions.fail("Failed to remove device data from gNMI", e);
         }
+
```

Comment:
Please add comment like: "Verify that device is removed"

Reply:
done